google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/features/InstanceApi.java

```diff
@@ -135,6 +135,17 @@ Operation deleteAccessConfigFromNic(@PathParam("instance") String instance,
    @GET
    @Path("/{instance}/serialPort")
    SerialPortOutput getSerialPortOutput(@PathParam("instance") String instance);
+   
+   /**
+    * Returns the specified instance's serial port output.
+    *
+    * @param instance the instance name.
+    * @return if successful, this method returns a SerialPortOutput containing the instance's serial output.
+    */
+   @Named("Instances:getSerialPortOutput") // FIXME What does "Named" really do?!
```

Comment:
It is used to allow to configure a specific timeout for this method. Users can set properties to customize timeouts for concrete methods by using the value in this annotation.